<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialLayer</class>
 <widget class="QScrollArea" name="MaterialLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer</string>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="layerList">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addLayerButton">
        <property name="text">
         <string>Add Layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteLayerButton">
        <property name="text">
         <string>Delete Layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveUpButton">
        <property name="text">
         <string>Move Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveDownButton">
        <property name="text">
         <string>Move Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="channelGroupBox">
      <property name="title">
       <string>Channel</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="colorBox">
         <property name="text">
          <string>Color</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="reflectionBox">
         <property name="text">
          <string>Reflection</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="glossnessBox">
         <property name="text">
          <string>Glossness</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="refractionBox">
         <property name="text">
          <string>Refraction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="iorBox">
         <property name="text">
          <string>IOR</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="thicknessBox">
         <property name="text">
          <string>Thickness</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="importChannelButton">
           <property name="text">
            <string>Import Image to Layer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportImageButton">
           <property name="text">
            <string>Export Layer to Image</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="exportFinalImageButton">
         <property name="text">
          <string>Export Canvas to Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
